packages/services/api/src/modules/admin/module.graphql.mappers.ts

```diff
@@ -1,10 +1,11 @@
+import type { Scalars } from '../../__generated__/types.next';
 import type { AdminOrganizationStats } from '../../shared/entities';
 
 export type AdminQueryMapper = {};
 export type AdminStatsMapper = {
   period: {
-    from: string;
-    to: string;
+    from: Scalars['DateTime']['output'];
+    to: Scalars['DateTime']['output'];
```

Comment:
This is required because I think we have been passing values for `DateTime` scalar slots as `Date`, but typed it as `string`.